app/uk/gov/hmrc/agentclientrelationshipsfrontend/views/client/ConfirmConsentPage.scala.html

```diff
@@ -99,9 +99,9 @@ <h2 class="govuk-heading-l">@messages(s"$key.h2")</h2>
                     classes = "govuk-fieldset__legend--m"
                 ))
             )),
-            items = Seq("true", "false").map(trueOrFalse =>
+            items = Seq("true", "false").zipWithIndex.map((trueOrFalse,index) =>
                 RadioItem(
-                    id = Some(ConfirmConsentFieldName),
+                    id = Some(s"$ConfirmConsentFieldName-$index"),
```

Comment:
the fix for the bug here is to remove the `id = Some(ConfirmConsentFieldName)` instead of replace it - .withFormField will automate the ids to be unique for each radio and on pattern as expected by error summary links